packages/convertkit-sdk/src/convertkit.ts

```diff
@@ -0,0 +1,216 @@
+import isEmpty from 'lodash/isEmpty'
```

Comment:
I pulled this out of skill-api (next) and added a form method. also added fallbacks for non-next env vars

Reply:
we'll want to use this new package in skill-api too for sure, just gonna do it in a future pr